.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: python
 python:
-  - "3.5"
-  - "3.5-dev"  # 3.5 development branch
   - "3.6"
-  - "3.6-dev"  # 3.6 development branch
```

Comment:
Why did you remove 3.6-dev and 3.7-dev?

Reply:
Ah. I thought we would not need to test for the dev branches if we are building against python 3.8.

Reply:
Will revert the change.